lib/provider/camunda/element-templates/parts/DescriptionProps.js

```diff
@@ -90,7 +91,9 @@ module.exports = function(
               eventDefinitionType: eventDefinitionType
             });
 
-            selection.select(newElement);
+            if (!is(newElement, 'bpmn:Process') && !is(newElement, 'bpmn:Collaboration')) {

```

Comment:
Why is this check necessary? What would we like to achieve? 

General remark: 

* From my point of view, this behavior in general could be part of the [`SelectionBehavior`](https://github.com/bpmn-io/diagram-js/blob/develop/lib/features/selection/SelectionBehavior.js) and not burried here, too.

Reply:
good point, see https://github.com/bpmn-io/diagram-js/pull/540/commits/33acbcf9cac50173a4487328580b58489ca2cf80